src/js/graphql_endpoint/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "allowJs": true,
+    "outDir": "./ts_compiled",
+
+    "lib": [
+      // no DOM
+      "ES5", 
+      "ScriptHost",
+    ],
+
+    //"strict": true,
+    //"downlevelIteration": true,
+    "noImplicitAny": true,
+    "forceConsistentCasingInFileNames": true,
+  },
+  "include": [
+    "server.ts",
+    "server.js",
+    "modules/**/*.js",
```

Comment:
Looks good, can you please test this locally, too? The FE still doesn't have great test coverage, and I manually an engagement just to be sure.

Reply:
Yep, totally can do